tsconfig.json

```diff
@@ -26,5 +22,14 @@
     "noUnusedLocals": true,
     "outDir": "./dist"
   },
-  "include": ["**/*.ts", "**/*.tsx", "**/*.d.ts", "examples/Example/index.js"]
+  "include": [
+    "src/**/*.ts",
+    "src/**/*.tsx",
+    "src/**/*.d.ts",
```

Comment:
To be removed too.

Reply:
This will be removed in the next PR.

Reply:
Just commenting for convenience along with other comments of removal.